modules/end-user-guide/partials/proc_using-an-embedded-editor-definition-for-your-ide.adoc

```diff
@@ -0,0 +1,52 @@
+[id="using-an-embedded-editor-ddefinition-for-your-ide_{context}"]
+=== Using an embedded editor definition for your IDE
+
+Fully customize the project IDE by using the `inline` directive to put a complete IDE definition in the `che-editor.yaml`` file 
```

Comment:
Consider providing more information about _why_ so the user can decide whether this choice makes sense for them. Red Hat encourages us to make opinionated recommendations, too, if they're justified.